moveit_planners/ompl/ompl_interface/src/parameterization/joint_space/joint_model_state_space_factory.cpp

```diff
@@ -52,5 +52,5 @@ int ompl_interface::JointModelStateSpaceFactory::canRepresentProblem(
 ompl_interface::ModelBasedStateSpacePtr
 ompl_interface::JointModelStateSpaceFactory::allocStateSpace(const ModelBasedStateSpaceSpecification& space_spec) const
 {
-  return ModelBasedStateSpacePtr(new JointModelStateSpace(space_spec));
+  return std::make_shared<JointModelStateSpace(space_spec);
```

Comment:
I think you meant this:
```
return std::make_shared<JointModelStateSpace>(space_spec)
```

Reply:
Oh right really sorry for this fixing asap.